src/main/java/duke/Tasks/Event.java

```diff
@@ -0,0 +1,87 @@
+package duke.Tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A Event class is a children object of the Task class
+ * with the additional abilities to
+ * returns date of the Event
+ * returns original command of the Event
+ */
```

Comment:
Perhaps, the language and punctuation can be improved. Such as period at the end of the sentences.